config/prod/dgutman-htan-s3.yaml

```diff
@@ -0,0 +1,45 @@
+# Provision a general S3 bucket or a Synapse External Bucket
+# http://docs.synapse.org/articles/custom_storage_location.html
+template_path: remote/s3-bucket.yaml
+stack_name: dgutman-htan-s3
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "HTAN"
+  # The resource owner
+  OwnerEmail: "xengie.doan@sagebase.org"
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+
+  # The following parameters are only examples they are not required.
+  # You may omit them if you do not need to override the defaults.
+
+  # (Optional) Synapse username (default ""), required if AllowWriteBucket=true
+  SynapseUserName: 'xengie.doan'
+  # (Optional) Allow accounts, groups, and users to access bucket. (default is no access).
+  GrantAccess:
+    - 'arn:aws:iam::325565585839:root'   # Required ARN for a synapse bucket
+    - 'arn:aws:iam::289701539771:dagutman'
+    - 'arn:aws:iam::563295687221:user/xengie.doan@sagebase.org'
+  # (Optional) true (default) to encrypt bucket, false for no encryption
```

Comment:
did you really want to override all the below params?  i'm thinking you probably just want the defaults